step.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -ex
+echo "Workflow id: ${workflow_id}"
+echo "Bitrise config path: ${bitrise_config_path}"
```

Comment:
please validate if `workflow_id` is empty
please validate if `bitrise_config_path ` is exist

Reply:
done